### PR TITLE
Adds modpack list verb

### PR DESCRIPTION
**Pull request by quardbreak**
изображение

### CODE REVIEW

File: mods/_modpack.dm [hunk 1]
```diff
@@ -1,16 +1,30 @@
...
+	var/list/credits_crew_names
+	/// A list of strings that are used by the end of round credits roll.
+	var/list/credits_holidays
+	/// A list of strings that are used by the end of round credits roll.
+	var/list/credits_adjectives
+	/// A list of strings that are used by the end of round credits roll.
+	var/list/credits_crew_outcomes
+	/// A list of strings that are used by the end of round credits roll.
+	var/list/credits_topics
+	/// A list of strings that are used by the end of round credits roll.
```
Comment (MistakeNot4892) [inline mods/_modpack.dm]: Why make the comments worse? :(

File: mods/_modpack.dm [hunk 2]
```diff
@@ -41,3 +55,33 @@
...
+						. += "<br>Description: [M.desc]"
+					if(M.author)
+						. += "<br><i>Author: [M.author]</i>"
+				. += "</div><br>"
+
+		var/datum/browser/popup = new(mob, "modpacks_list", "Modpacks List", 480, 580)
+		popup.set_content(.)
+		popup.open()
+	else
+		to_chat(src, SPAN_WARNING("This server doesn't have any available modpacks."))
```
Comment (MistakeNot4892) [inline mods/_modpack.dm]: 'Does not include any modpacks' would read better imo.

File: mods/_modpack.dm (hunk 1 above)
Comment (quardbreak) [reply mods/_modpack.dm]: I did it to respect dmdoc :^(

Comment (MistakeNot4892) [reply mods/_modpack.dm]: As Lohi said you can use `// !` instead of making it untidy.

**Review by MistakeNot4892 — APPROVED** [review]

**Pull request merged**